firmware/smbios/bios_information.ksy

```diff
@@ -0,0 +1,42 @@
+meta:
+  id: bios_information
```

Comment:
Please note that we effectively have a flat ID name space, so this `bios_information` will likely clash with other formats. Please consider being more specific and using some kind of a prefix, such as `smbios_bios_info`.

Reply:
Changed! but still having tests failed on CircleCI.

Reply:
Thanks! This is actually a very good question why these CircleCI jobs get to run and what it is attempting to do. There should be no PR-related jobs running at the CI in this repo, to my knowledge.